docs/compose-file.md

```diff
@@ -580,11 +580,16 @@ should always begin with `.` or `..`.
       # Named volume
       - datavolume:/var/lib/mysql
 
-If you use a volume name (instead of a volume path), you may also specify
-a `volume_driver`.
+If you do not use a host path, you may specify a `volume_driver`.
 
     volume_driver: mydriver
 
+Note that for [version 2 files](#version-2), this driver
+will not apply to named volumes (you should use the `driver` option when
+[declaring the volume](#volume-configuration-reference) instead).
+For [version 1](#version-1), both named volumes and container volumes will
+be afected.
```

Comment:
Typo: "afected". But actually, I think better wording would be: "For version 1, both named volumes and container volumes will use the specified driver".